manifests/init.pp

```diff
@@ -9,6 +9,7 @@
   $enable_fqdn_entry     = true,
   $use_fqdn              = true,
   $fqdn_host_aliases     = $::hostname,
+  $fqdn_ip_address       = $::ipaddress,
```

Comment:
This new parameter needs to be listed in the README and validated with `is_ip_address()`

https://github.com/puppetlabs/puppetlabs-stdlib/tree/4.6.0#is_ip_address